velox/expression/SimpleFunctionAdapter.h

```diff
@@ -405,13 +372,40 @@ class SimpleFunctionAdapter : public VectorFunction {
 
   template <
       int32_t POSITION,
-      bool primitiveFlatConstantFastPath,
+      typename Func1,
```

Comment:
nit: perhaps, FMakeReader and FNextNotNull for clarity?

Reply:
I removed those, updated the diff